tests/queries/0_stateless/03202_pretty_truncate_column_name.sql

```diff
@@ -0,0 +1,13 @@
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettyCompact;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettyCompactNoEscapes;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettyCompactMonoBlock;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettyCompactNoEscapesMonoBlock;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettySpace;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettySpaceNoEscapes;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettySpaceMonoBlock;
+select 3760925978781085988 as "sum(cityHash64(article, byline, dates, newspaper_metadata, antitrust, civil_rights, crime, govt_regulation, labor_movement, politics, protests, ca_topic, ner_words, ner_labels, wire_city, wire_state, wire_country, wire_coordinates, wire_location_notes, cluster_size, year))" FORMAT PrettySpaceNoEscapesMonoBlock;
```

Comment:
It's also would be good to add a test case when we change the settings you've created (worth trying to set it to some normal values, 0 and negative values)